orchestration/hca_orchestration/repositories/base_repositories.py

```diff
@@ -36,14 +36,11 @@ def copy_project_to_new_dataset_job() -> PipelineDefinition:
         })
 
 
-def base_jobs() -> list[Union[PipelineDefinition, SensorDefinition]]:
+def base_jobs(mode: str) -> list[Union[PipelineDefinition, SensorDefinition]]:
     defs = [
         cut_snapshot,
         load_hca,
-        validate_egress,
-        build_post_import_sensor(os.environ.get("ENV", "test")),
-        copy_project_to_new_dataset_job(),
+        validate_egress
     ]
 
-    return defs + load_dcp_release_manifests() + copy_project_to_new_dataset_partitions() + \
-        dev_refresh_cut_snapshot_partition_set()
```

Comment:
Same comments here, copy_project_to_new_dataset_partitions and dev_refresh_cut_snapshot_partition_set only make sense in dev.